### DISCUSSION TITLE
Translation

### DISCUSSION THREAD

**Discussion by ghost** [fexh10/aw-cli#5] [Q&A] [answered]
Would it be possible for someone who can speak both English and the language this is programmed in to translate some of the documentation over?

**Comment by axtrat** [comment, marked as answer]
Yes, it would be technically possible, but there is no point in doing it.
Our project is more or less an Italian adaptation of [ani-cli](https://github.com/pystardust/ani-cli): "A cli to browse and watch anime" in English (implemented in bash), with some divergences from the original to implement features that were relevant to us.
If you are looking for a Python-based alternative there is also [anipy-cli](https://github.com/sdaqo/anipy-cli): "ani-cli rewritten in python"
